plugins/ci/pkg/ci/polaris.go

```diff
@@ -9,22 +10,22 @@ import (
 	"github.com/fairwindsops/insights-plugins/plugins/ci/pkg/models"
 )
 
-func (ci *CIScan) GetPolarisReport() (models.ReportInfo, error) {
+func (ci *CIScan) GetPolarisReport() (*models.ReportInfo, error) {
 	report := models.ReportInfo{
 		Report:   "polaris",
 		Filename: "polaris.json",
 	}
-	// Scan with Polaris
-	_, err := commands.ExecWithMessage(exec.Command("polaris", "audit", "--audit-path", ci.configFolder, "--output-file", filepath.Join(ci.config.Options.TempFolder, report.Filename)), "Audit with Polaris")
-	if err != nil {
-		return report, err
-	}
 	polarisVersion, err := commands.Exec("polaris", "version")
 	if err != nil {
-		return report, err
+		return nil, fmt.Errorf("unable to get polaris version: %v: %v", err, polarisVersion)
 	}
 	report.Version = strings.Split(polarisVersion, ":")[1]
-	return report, nil
+	// Scan with Polaris
```

Comment:
This comment is probably unnecessary. Great use of comments overall though!

Reply:
That comment was part of the original code and I left it because I wasn't sure if it was useful to someone.